Makefile

```diff
@@ -105,8 +105,10 @@ else
 IS_CYGWIN := $(shell $(CC) -dumpmachine | grep -i cygwin | wc -l)
 ifeq ($(IS_CYGWIN),1)
 EXT = dll
-AR_EXT = lib
+AR_EXT = a
```

Comment:
cyg*.dll and lib*.a are used for Cygwin but not for native Windows.
Cygwin's gcc can link these, and also Cygwin's python can load them.
If you want to generate a library which can be used for native Windows, you should compile for MinGW.